app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,134 @@
+class Api::V1::UsersController < ApiController
+
+  include Swagger::Blocks
+
+  swagger_path '/users' do
+    operation :get do
+      key :summary, 'Get all users for the membership'
+      key :description, 'List all users'
+      key :operationId, 'getUsers'
+      security do
+        key :api_token, []
+      end
+      key :tags, ['Users']
+
+      extend Api::SwaggerResponses
+      include_404_schema
+      include_success_schema(model: 'User', format: :array)
+    end
+  end
+  def index
+    render api_response data: current_member.users
+  end
+
+  swagger_path '/users/add' do
+    operation :post do
+      key :summary, 'add a newly created user account to a membership'
+      key :operationId, 'claimUser'
+      security do
+        key :api_token, []
+      end
+      key :tags, ['Users']
+      parameter do
+        key :name, :email
+        key :in, :path
+        key :description, 'email of user to join to membership'
+        key :required, true
+      end
+      parameter do
+        key :name, :role
+        key :in, :body
+        key :description, 'role to grant to user'
+        key :required, true
+      end
+      extend Api::SwaggerResponses
+      include_404_schema
```

Comment:
Don't see where a 404 or 406 is generated below

Reply:
good point, add will be a `not_acceptable` from  `render_invalid_response` or it'll send the invite and be a 200